src/components/NinDisplay.js

```diff
@@ -18,15 +45,26 @@ export class NinDisplay extends Component {
           {this.props.translate("nin_display.profile.no_nin")}
         </Link>,
       ];
-    } else {
-      if (this.props.verifiedNinStatus) {
-        userData = [
-          <div key="1" data-ninnumber={this.props.verifiedNin[0].number}>
-            <p key="0" className="display-data verified">
-              {this.props.verifiedNin[0].number}
-            </p>
-          </div>,
-        ];
+    }else{
+      if(this.props.verifiedNinStatus) {
```

Comment:
Maybe I am missing something obvious and maybe it can be explained through naming of variables, but it is unclear why verified status is a condition for this render. It is important for the number and toggle button to know if the number is verified or not?